tests/test_lib.py

```diff
@@ -198,26 +198,19 @@ def tearDown_test_db():
     """Deletes the test db
         although this seams not to work on my system it leaves me with an zero kb file
     """
-    # uncomment next line so leave the db intact between test and at the end
-    # return False
 
-    try:
-        if os.path.exists(os.path.join(TESTDIR, TESTDBNAME)):
-            os.remove(os.path.join(TESTDIR, TESTDBNAME))
-    except:
-        pass
+    #uncomment next line so leave the db intact between test and at the end
+    #return False
 
-    try:
-        if os.path.exists(os.path.join(TESTDIR, TESTCACHEDBNAME)):
-            os.remove(os.path.join(TESTDIR, TESTCACHEDBNAME))
-    except:
-        pass
+    for current_db in [ TESTDBNAME, TESTCACHEDBNAME, TESTFAILEDDBNAME ]:
+        for file_name in [ os.path.join(TESTDIR, current_db), os.path.join(TESTDIR, current_db + '-journal') ]:
+            if os.path.exists(file_name):
+                try:
+                    os.remove(file_name)
+                except IOError, e:
```

Comment:
This should check for IOError and OSError and be in the new format( except (IOError, OSError) as e)